UnitySDK/Assets/ML-Agents/Scripts/Academy.cs

```diff
@@ -138,10 +168,60 @@ public void LazyInitialization()
             }
         }
 
+        /// <summary>
+        /// Enable stepping of the Academy during the FixedUpdate phase.  This is done by creating a temporary
+        /// GameObject with a MonoBehavior that calls Academy.EnvironmentStep().
+        /// </summary>
+        public void EnableAutomaticStepping()
+        {
+            if (m_FixedUpdateStepper != null)
+            {
+                return;
+            }
+
+            m_StepperObject = new GameObject("AcademyFixedUpdateStepper");
+            // Don't show this object in the hierarchy
+            m_StepperObject.hideFlags = HideFlags.HideInHierarchy;
+            m_FixedUpdateStepper  = m_StepperObject.AddComponent<AcademyFixedUpdateStepper>();
+            m_FixedUpdateStepper = m_StepperObject.GetComponent<AcademyFixedUpdateStepper>();
```

Comment:
```suggestion

```

Reply:
can remove this line

Reply:
I deleted the good line :(  Gave up doing that from github UI and fixed it for real.